htdocs/core/class/commonobject.class.php

```diff
@@ -3715,10 +3715,19 @@ function formAddObjectLine($dateSelector, $seller, $buyer)
 
 		// Output template part (modules that overwrite templates must declare this into descriptor)
 		// Use global variables + $dateSelector + $seller and $buyer
-		$dirtpls=array_merge($conf->modules_parts['tpl'],array('/core/tpl'));
-		foreach($dirtpls as $reldir)
+		$dirtpls=array_merge($conf->modules_parts['tpl'], array('/core/tpl'));
+		foreach($dirtpls as $module => $reldir)
 		{
-			$tpl = dol_buildpath($reldir.'/objectline_create.tpl.php');
+			if (!empty($module) && empty($conf->$module->enabled)) continue;
```

Comment:
If a module was disabled, the $conf->modules_parts['tpl'] does not contains the entry for module so the test is useless here.
If you have an entry $conf->modules_parts['tpl'] when module is disabled, try to run repair.php to clean you llx_const table.